telegram/utils/helpers.py

```diff
@@ -72,3 +73,45 @@ def from_timestamp(unixtime):
         return None
 
     return datetime.fromtimestamp(unixtime)
+
+
+def mention(user, html=False, name=None):
+    """
+    Args:
+        user (:obj:`int` | :class:`telegram.User` | :class:`telegram.Message`)
+            The user's id, or User object, or a Message from the user which you want to mention.
+        html (:obj:`bool`) Default output as markdown, pass True to use HTML.
+        name (:obj:`str`) The name the mention is showing. Required when id is provided in user.
+            Optional if User or Message is provided and will overwrite the name in user.
+
+    Returns:
+        str:
```

Comment:
Please define return as :obj :`str`: The inline mention for the user as html or markdown.